label_studio/projects/functions/__init__.py

```diff
@@ -22,8 +22,8 @@ def annotate_finished_task_number(queryset):
 
 
 def annotate_total_predictions_number(queryset):
-    return queryset.annotate(total_predictions_number=Count('tasks__predictions', distinct=True))
-
+    predictions = Prediction.objects.filter(task__project=OuterRef('id')).values('id')
```

Comment:
another thing for optimization: add project field to predictions. This should boost the whole our app very well. But it's another ticket.

Reply:
ah right it would remove that extra join through the task.  Sounds like there's another ticket for it already? but if not I can create one.